### ISSUE TITLE
Security & Nullable fields

### ISSUE THREAD

**Issue by Vorian-Atreides** [go-swagger/go-swagger#1546]
## Problem statement

I encountered several issues related to swagger 'security' definitions and nullable fields.
Some of them have resolved issues and make me wonder if they aren't a regression.

## Swagger specification

2.0

## Steps to reproduce

Missing fields:
```
security: []
authorizationUrl: ""
```

Both of these fields will be removed in the generated spec, either configured as a comment or from an existing swagger.yaml.

----

Ignored combination:
```
security:
  - apiKey
     oauth2

security:
   - apiKey
   - oauth2
```
The first define an "AND" condition while the second define an "OR". At the moment, defining the first will generate an "OR".

## Environment
swagger version: 0.13.0 
go version: 1.10 
OS:   OS X

**Comment by gregmarr** [comment]
These issues were resolved after 0.13.0

**Comment by casualjim** [comment]
0.14 was released
